### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by vikshith7**
design with MVVM and API Integration with out connections.
Changes TODO comments to make API Integrations

### CODE REVIEW

File: .idea/render.experimental.xml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (geethukk) [inline .idea/render.experimental.xml]: Always add .idea folder and .iml file to .gitignore

File: app/src/main/java/com/ukpc/ipark/customWidgets/FooterLayout.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import androidx.databinding.BindingAdapter;
+import androidx.databinding.DataBindingComponent;
+import androidx.databinding.DataBindingUtil;
+
+import com.ukpc.ipark.R;
+import com.ukpc.ipark.databinding.FooterLayoutBinding;
+
+public class FooterLayout extends LinearLayout implements DataBindingComponent {
+    final private Context mContext;
+    //    private AppCompatImageView car1, car2, car3;
```
Comment (geethukk) [inline app/src/main/java/com/ukpc/ipark/customWidgets/FooterLayout.java]: Please remove commented codes

File: app/src/main/java/com/ukpc/ipark/mvvm/model/RegistrationModel.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    private String userId;
+    private String maxDays;
+    private String recordTime;
+    private String verified;
+    private String syncStatus;
+
+    public RegistrationModel() {
+    }
+
+    public RegistrationModel(String registrationNo, String userId, String maxDays, String recordTime, String verified, String syncStatus) {
```
Comment (geethukk) [inline app/src/main/java/com/ukpc/ipark/mvvm/model/RegistrationModel.java]: format all the codes and then save

File: app/src/main/java/com/ukpc/ipark/mvvm/viewModel/LoginActViewModel.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    public ApiTaskHandlers apiTaskHandlers;
+    LoginRepository loginRepository;
+
+    public LoginActViewModel() {
+    }
+
+    public boolean isValidEmail() {
+        if (userEmail != null)
+            // todo change comments
+//            return userEmail.trim().length() != 0;
```
Comment (geethukk) [inline app/src/main/java/com/ukpc/ipark/mvvm/viewModel/LoginActViewModel.java]: Move hardcoded strings to a constant file

File: app/src/main/java/com/ukpc/ipark/ui/activity/LoginActivity.java [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+                if (loginActViewModel.isValidEmail() && loginActViewModel.isValidPassword()) {
+                   /*
+                    todo change comments
+                    CommonFun.showProgressDialog(LoginActivity.this);
+                    loginActViewModel.checkLoginFromServer().observe(this, new Observer<BaseResponse>() {
+                        @Override
+                        public void onChanged(BaseResponse baseResponse) {
+                            CommonFun.dismissProgressDialog();
+
+                            if (baseResponse.isInternet() || baseResponse.isApiUnSuccessful()) {
```
Comment (geethukk) [inline app/src/main/java/com/ukpc/ipark/ui/activity/LoginActivity.java]: Please remove unwanted codes if not needed

File: app/src/main/java/com/ukpc/ipark/ui/activity/MainActivity.java [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+    public String model = "AUDI", make = "SQ5 TFSI QUATTRO AUTO", color = "WHITE";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        activityMainBinding = DataBindingUtil.setContentView(this, R.layout.activity_main);
+
+        adapter = new ViewpagerAdapter(getSupportFragmentManager());
+
+        CarRegistrationFrag carRegistrationFrag = new CarRegistrationFrag();
```
Comment (geethukk) [inline app/src/main/java/com/ukpc/ipark/ui/activity/MainActivity.java]: Move hardcoded strings to a constant file

File: app/src/main/java/com/ukpc/ipark/ui/activity/SplashActivity.java [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+        getWindow().getDecorView().setSystemUiVisibility(
+                View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN | View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION);
+
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash);
+//        ipark_imageView = findViewById(R.id.ipark_imageView);
+//        ukpc_imageView = findViewById(R.id.ukpc_imageView);
+        /*ukpc_imageView.setVisibility(View.INVISIBLE);
+        Animation animationSet = AnimationUtils.loadAnimation(SplashActivity.this, R.anim.zoom_in);
+        ipark_imageView.setAnimation(animationSet);
```
Comment (geethukk) [inline app/src/main/java/com/ukpc/ipark/ui/activity/SplashActivity.java]: Please remove commented codes(which is not using ) in whole repository

File: app/src/main/java/com/ukpc/ipark/customWidgets/FooterLayout.java (hunk 2 above)
Comment (vikshith7) [reply app/src/main/java/com/ukpc/ipark/customWidgets/FooterLayout.java]: done.

File: app/src/main/java/com/ukpc/ipark/mvvm/model/RegistrationModel.java (hunk 3 above)
Comment (vikshith7) [reply app/src/main/java/com/ukpc/ipark/mvvm/model/RegistrationModel.java]: already in format.

File: app/src/main/java/com/ukpc/ipark/mvvm/viewModel/LoginActViewModel.java (hunk 4 above)
Comment (vikshith7) [reply app/src/main/java/com/ukpc/ipark/mvvm/viewModel/LoginActViewModel.java]: done.

File: app/src/main/java/com/ukpc/ipark/ui/activity/LoginActivity.java (hunk 5 above)
Comment (vikshith7) [reply app/src/main/java/com/ukpc/ipark/ui/activity/LoginActivity.java]: done.

File: app/src/main/java/com/ukpc/ipark/ui/activity/MainActivity.java (hunk 6 above)
Comment (vikshith7) [reply app/src/main/java/com/ukpc/ipark/ui/activity/MainActivity.java]: done.

File: app/src/main/java/com/ukpc/ipark/ui/activity/SplashActivity.java (hunk 7 above)
Comment (vikshith7) [reply app/src/main/java/com/ukpc/ipark/ui/activity/SplashActivity.java]: done.

**Pull request closed without merging**